File: Advanced Methods of Programming/Toy Language Interpreter/src/Model/Commands/Command.java
package Model.Commands;

public abstract class Command {
    private String key;
    private String description;

    public Command(String key, String description) {
        this.key = key;
        this.description = description;
    }

    public abstract void execute();

    public String getKey(){
        return key;
    }

    public String getDescription(){
        return description;
    }
}

File: Advanced Methods of Programming/CSharp Toy Language Interpreter/ToyLanguageInterpreter/Src/Model/Commands/Command.cs

namespace ToyLanguageInterpreter.Src.Model.Commands
{
    abstract class Command
    {
        public string Key { get; }
        public string Description { get; }

        public Command(string key, string description)
        {
            Key = key;
            Description = description;
        }

        public abstract void Execute();
    }
}
